src/OpenTelemetry.Instrumentation.EntityFrameworkCore/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+* Updated OTel SDK package version to 1.4.0-rc.4
```

Comment:
could you update readme.md file to show the new capabilities this PR enables which was previously unavailable?

Reply:
You could use this PR's CHANGELOG updates for reference: https://github.com/open-telemetry/opentelemetry-dotnet/pull/3664/files#diff-4f436e3acc6e1645db5b52fff0eb65303bad34c84466f1461005dc0040f23232R8-R10

Reply:
Let's finish the review on the PR #1019. Then I'll reflect the same changes in this PR.